src/server/pathway-commons/index.js

```diff
@@ -44,14 +44,14 @@ const _processQueryString = async (inputString) => {
   ];
 };
 
-//Pathway Commons HTTP GET request; options.cmd = 'get', 'search', 'traverse', 'graph', etc.
+//Pathway Commons HTTP GET request; options.cmd = 'pc2/get', 'pc2/search', 'pc2/traverse', 'pc2/graph', etc.
 const query = async (queryObj) => {
```

Comment:
Is it possible to remove any mention of `'pc2'` especially in the client (e.g. `ServerAPI.pcQuery('pc2/get',`) in favour of appending the 'pc2' here? This would require a change to router for `'/pc2/:path'` to use `query`

Reply:
no